_compress_images_cache.yml

```diff
@@ -2200,3 +2200,32 @@ assets/images/guide/resources/design-files/design-source-files-preview@2x.jpg: d
 assets/images/guide/resources/design-files/design-source-files.jpg: design-source-files.jpg
 assets/images/guide/resources/design-files/design-source-files-mobile.jpg: design-source-files-mobile.jpg
 assets/images/guide/resources/design-files/design-source-files-mobile@2x.jpg: design-source-files-mobile@2x.jpg
+assets/images/guide/contribute/case-studies/CaseStudiesHeader2X.png: CaseStudiesHeader2X.png
+assets/images/guide/contribute/case-studies/CaseStudiesHeaderMobile.png: CaseStudiesHeaderMobile.png
+assets/images/guide/contribute/case-studies/CasestudiesHeader.png: CasestudiesHeader.png
+assets/images/guide/contribute/case-studies/CasestudiesHeaderMobile2X.png: CasestudiesHeaderMobile2X.png
+assets/images/guide/contribute/case-studies/UserPersonasExampleBlixt.png: UserPersonasExampleBlixt.png
+assets/images/guide/contribute/case-studies/UserPersonasExampleBlixtMobile.png: UserPersonasExampleBlixtMobile.png
+assets/images/guide/contribute/case-studies/UserPersonasExampleBlixtMobileX2.png: UserPersonasExampleBlixtMobileX2.png
+assets/images/guide/contribute/case-studies/UserPersonasExampleBlixtX2.png: UserPersonasExampleBlixtX2.png
+<<<<<<< Updated upstream
```

Comment:
Your preview site is not building because there are unresolved merge issues in this image cache file. Remove lines 2211, 2212, and 2231.